spec/factories/pets.rb

```diff
@@ -8,4 +8,22 @@
       sequence(:description) { |n| "Only has #{n} diseases!" }
       shelter
   end
+  factory :pet1 do
+    name {"Dr. Dog"}
+    age {34}
+    sex {"male"}
```

Comment:
this wasnt exactly working for me. but ill find it

Reply:
Dr. Dog is a great band